app/src/main/java/ch/epfl/sdp/drone3d/map/offline/OfflineMapSaverImpl.kt

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+package ch.epfl.sdp.drone3d.map.offline
+
+import android.content.Context
+import android.util.Log
```

Comment:
Try to think to format & clean your files, you have unused import here.

Reply:
My bad, corrected